authentik/policies/geoip/api.py

```diff
@@ -0,0 +1,49 @@
+"""GeoIP Policy API Views"""
+
+from django_countries.serializer_fields import CountryField
+from drf_spectacular.utils import extend_schema_field, inline_serializer
+from rest_framework import serializers
+from rest_framework.viewsets import ModelViewSet
+
+from authentik.core.api.used_by import UsedByMixin
+from authentik.policies.api.policies import PolicySerializer
+from authentik.policies.geoip.models import GeoIPPolicy
+
+COUNTRY_SCHEMA = {
+    "code": CountryField(),
+    "name": serializers.CharField(),
+}
```

Comment:
This is an exact copy of `authentik/policies/geoip/views:11-14`. Weirdly, importing it from there results in `gen-build` silently failing to generate this part of the schema. I'm out of my depth here, so I just duplicated these 4 lines.

Reply:
That is very odd but also doesn't surprise me